cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/GeneratorConversionTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2024, Artelys (http://www.artelys.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.cgmes.conversion.test;
+
+import com.powsybl.cgmes.conversion.Conversion;
+import com.powsybl.commons.datasource.ResourceDataSource;
+import com.powsybl.commons.datasource.ResourceSet;
+import com.powsybl.iidm.network.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Damien Jeandemange {@literal <damien.jeandemange at artelys.com>}
+ */
+class GeneratorConversionTest {
+
+    @Test
+    void generatingUnitTypes() {
+        Network network = Network.read(new ResourceDataSource("GeneratingUnitTypes", new ResourceSet("/", "GeneratingUnitTypes.xml")));
```

Comment:
A new method has been added recently to facilitate the reading of CGMES resources, which is often being performed in CGMES conversion tests. The method is: ```readCgmesResources``` and can be imported as static from ```ConversionUtil```

Reply:
thanks, done